src/components/mediaViewer/mediaViewer.jsx

```diff
@@ -34,17 +34,17 @@ export function MediaViewer({ src, type, alt, thumbnail }) {
   return (
     <BrowserOnly>
       {() => {
-        const thumbnailSrc = getSrc(thumbnail || src);
         let contentSrc = getSrc(src);
+        const thumbnailSrc = thumbnail ? getSrc(thumbnail) : contentSrc;
         const isVideo = type === TYPE_VIDEO;
 
-        if (isVideo) {
+        if (isVideo && contentSrc.includes('youtu.be/')) {
           contentSrc = contentSrc.replace('.be/', 'be.com/embed/');
         }
 
         return (
           <>
-            <div className={`${!isVideo ? 'media-container' : 'video-container'}` } onClick={() => setOpen(true)} >
+            <div className={`${isVideo ? 'video-container' : 'media-container'}`} onClick={() => setOpen(true)} >
```

Comment:
Unnecessary string template, can be just `isVideo ? 'video-container' : 'media-container'`